front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/V2PipelineTemplateController.java

```diff
@@ -70,9 +71,14 @@
 
   // TODO(jacobkiefer): Add fiat authz
   @RequestMapping(value = "", method = RequestMethod.GET)
-  List<PipelineTemplate> list(
+  Map<String, List<PipelineTemplate>> list(
```

Comment:
In general, I wouldn't encourage changing an API's contract like this. It means you have to upgrade `front50`, `gate`, `deck`, and `spin` all at the same time. If any one of them gets out of sync, you'll get 500 errors, since callers receive a map, but try to read it as a list (or vice versa).

Usually I'd say you should add a new API in `front50` and `gate`, then change `deck` and `spin` to call that new API and then later remove the old API. But maybe that level of safety isn't something we care about since probably only Netflix upgrades things separately?

Reply:
It's a good point that users may be upgrading components separately. I'll put this into a new endpoint with a deprecation plan for the original endpoint.